From khali@linux-fr.org Thu Oct  6 13:46:25 2005
Date: Wed, 21 Sep 2005 22:26:31 +0200
From: Jean Delvare <khali@linux-fr.org>
To: Greg Kroah-Hartman <gregkh@suse.de>
Message-Id: <20050921222631.453b7117.khali@linux-fr.org>
Subject: [PATCH 2.6] PCI: Add quirk for SMBus on HP D530

One more SMBus unhiding quirk, this time for the HP D530. Requested and
successfully tested by Ben Cranston.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/pci/quirks.c |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/pci/quirks.c
+++ gregkh-2.6/drivers/pci/quirks.c
@@ -950,6 +950,7 @@ static void __init asus_hides_smbus_host
 		if (dev->device == PCI_DEVICE_ID_INTEL_82865_HB)
 			switch (dev->subsystem_device) {
 			case 0x12bc: /* HP D330L */
+			case 0x12bd: /* HP D530 */
 				asus_hides_smbus = 1;
 			}
 	} else if (unlikely(dev->subsystem_vendor == PCI_VENDOR_ID_TOSHIBA)) {
